docker/coturn/turnserver.conf

```diff
@@ -723,3 +766,31 @@ cli-password=CHANGE_ME
 #no-tlsv1
 #no-tlsv1_1
 #no-tlsv1_2
+
+# Disable RFC5780 (NAT behavior discovery).
+#
+# Originally, if there are more than one listener address from the same
+# address family, then by default the NAT behavior discovery feature enabled.
+# This option disables the original behavior, because the NAT behavior
+# discovery adds extra attributes to response, and this increase the
+# possibility of an amplification attack.
+#
+# Strongly encouraged to use this option to decrease gain factor in STUN
+# binding responses.
+#
+no-rfc5780
```

Comment:
@ggarber this changes the default setting for docker containers
I do not think this is bad (better defaults - more "secure") - but it changes the default
Wanted to have another 👀 on it

Reply:
I think it is fine.    I think the docker images are new enough that the people using rfc5780 are probably not using them.

I would land it and include it in the "Release noteS".  I'll try to take anote.